Formula/m/mingw-w64.rb

```diff
@@ -22,13 +22,17 @@ class MingwW64 < Formula
     sha256 x86_64_linux:   "18f643336db02fa5262279462b07f8150e3fc5e02583ac8d5d7e331b5e82561f"
   end
 
+  depends_on "pkg-config" => :build
```

Comment:
Where is this used? I think neither binutils nor gcc use pkg-config.

Reply:
Binutils uses pkg-config to search for zstd (see: https://sourceware.org/git/?p=binutils-gdb.git;a=blob;f=binutils/configure;h=5d87b6b3e665776c816e6d77fd94dc43e056a469;hb=HEAD#l15795 ).

Would you like a comment to be added to clarify that ?